src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,122 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int USER_OVER_MAX_AGE = 125;
+    private static final int USER_UNDER_MIN_AGE = 17;
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+    private static User user;
+
+    @BeforeAll
+    public static void beforeAll() {
+        storageDao = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    private User defaultUser2Supplier() {
+        User user2 = new User();
```

Comment:
```suggestion
        User anotherUser = new User();
```
using numbers in names is a bad practice